### PR TITLE
Bug report template improvements

### PR DESCRIPTION
**Pull request by mfazekas**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -29,10 +29,53 @@ A clear and concise description of what the bug is.
 **To Reproduce**
 Steps to reproduce the behavior. 
 
-If possible include a single standalone React Native component. Use [one of our example](https://github.com/react-native-mapbox-gl/maps/blob/master/example/src/examples/PointInMapView.js) screens as a starging point.
+Please include a single standalone React Native component. Use [one of our example](https://github.com/react-native-mapbox-gl/maps/blob/master/example/src/examples/BugReportTemplate.js) screens as a starging point.
```
Comment (ferdicus) [inline .github/ISSUE_TEMPLATE/bug_report.md:32]: typo: `starging point` -> starting point

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -29,10 +29,53 @@ A clear and concise description of what the bug is.
...
+Please simplify the exampel as much as possible.
```
Comment (ferdicus) [inline .github/ISSUE_TEMPLATE/bug_report.md:33]: typo: `exampel` -> example

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (mfazekas) [inline .github/ISSUE_TEMPLATE/bug_report.md:32]:
```suggestion
Please include a single standalone React Native component. Use [one of our example](https://github.com/react-native-mapbox-gl/maps/blob/master/example/src/examples/BugReportTemplate.js) screens as a starting point.
```

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (mfazekas) [inline .github/ISSUE_TEMPLATE/bug_report.md:33]:
```suggestion
Please simplify the example as much as possible.
```

**Comment by mfazekas** [comment]
> Cool, thanks for this 👍
> 
> I might change the layout a bit in the future.
> 
> Another thing I was thinking about - maybe we could utilize GH Actions a bit more in our repo for ticket verification? Among other things.

Sure that would be great to have automated ticket verification

**Pull request merged**